drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -1029,10 +1033,13 @@ protected void checkValidAndUniquePropertyHeaderTitle(String propertyHeaderCellV
      * Verify if the given value is not already used as property header name <b>inside the same instance</b>
      * @param propertyHeaderCellValue
      * @param columnIndex
-     * @throws Exception if the given <b>propertyHeaderCellValue</b> has already been used
+     * @throws IllegalArgumentException if the given <b>propertyHeaderCellValue</b> has already been used
      * inside the <b>instance</b> of the given column
      */
-    protected void checkUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) throws Exception {
+    protected void checkUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) {
+        if (Objects.isNull(propertyHeaderCellValue) || propertyHeaderCellValue.isEmpty()) {
```

Comment:
We use this condition so often, I proposed in some PRs to use `org.drools.core.util.StringUtils.isEmpty(propertyHeaderCellValue)`

Reply:
@jomarko Thank you for the hint, done!

Reply:
@jomarko I restored my original code because org.drools.core.util.StringUtils.isEmpty is intended to be used in backend code. Unfortunately, I can't find a similar StringUtils class in frontend side.

Reply:
:( oh, ok,was there some dependency problem?

Reply:
@jomarko Yes, as you can see here:  https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/drools-wb-pullrequests/262/ . It's possibile to use it, but: you have to import drools-core in the frontend module. Anyway I don't think is a good idea, I checked where it's currently used and I found it ONLY in backend code.

Reply:
@yesamer sounds reasonable, thanks for investigation, please let me know once you have a look on my manual review notes.